CONTRIBUTING.md

```diff
@@ -43,3 +46,39 @@ make test-integration
 * Commits should be as small as possible, while ensuring that each commit is correct independently.
 
 * Add tests relevant to the fixed bug or new feature.
+
+
+## Troubleshooting build
+
+### docker approach
+you can build in a docker container by going to root dir of gopass project and:
```

Comment:
Capitalization, missing article confusing use of project. (`the root dir of your gopass checkout`).